core/gui/src/app/workspace/component/workflow-editor/workflow-editor.component.ts

```diff
@@ -426,16 +428,29 @@ export class WorkflowEditorComponent implements AfterViewInit, OnDestroy {
     fromJointPaperEvent(this.paper, "cell:pointerdblclick")
       .pipe(untilDestroyed(this))
       .subscribe(event => {
-        const clickedCommentBox = event[0].model;
+        const clickedBox = event[0].model;
         if (
-          clickedCommentBox.isElement() &&
-          this.workflowActionService.getTexeraGraph().hasCommentBox(clickedCommentBox.id.toString())
+          clickedBox.isElement() &&
+          this.workflowActionService.getTexeraGraph().hasCommentBox(clickedBox.id.toString())
         ) {
           this.wrapper.setMultiSelectMode(<boolean>event[1].shiftKey);
           const elementID = event[0].model.id.toString();
           if (this.workflowActionService.getTexeraGraph().hasCommentBox(elementID)) {
             this.openCommentBox(elementID);
           }
+        } else if (
+          clickedBox.isElement() &&
+          this.workflowActionService.getTexeraGraph().hasOperator(clickedBox.id.toString())
+        ) {
+          this.wrapper.setMultiSelectMode(<boolean>event[1].shiftKey);
+          const elementID = event[0].model.id.toString();
+          if (this.workflowActionService.getTexeraGraph().hasOperator(elementID)) {
+            if (this.panelservice.isPanelOpen) {
```

Comment:
you can merge this `if-else` block to a method called `this.panelservice.togglePropertyPanel()`

Reply:
also make sure to use the name `panelService` (camel case)